scripts/feature_tests/upgrade/controlplane_components_upgrade/upgrade_CAPI_and_CAPM3_with_clusterctl.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -x
+
+METAL3_DEV_ENV_DIR="$(dirname "$(readlink -f "${0}")")/../../../../"
+
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+source "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade/common.sh"
```

Comment:
Could you please rename common.sh to something that doesn't interfere with lib/common.sh?

Reply:
renamed

Reply:
> renamed

@Xenwar are you sure it is renamed? If `common.sh` is the concern, we could maybe rename it to `upgrade_common.sh`? Just a thought.

Reply:
Quite strange. I thought  I renamaed the common.sh file and all the references. 
Now, I see that is not the case. Need to check again.

Reply:
Pushed the new code. I can see the changes now.